lib/krane/deprecated_deploy_task.rb

```diff
@@ -597,7 +597,11 @@ def ejson_keys_secret
     end
 
     def statsd_tags
-      %W(namespace:#{@namespace} sha:#{@current_sha} context:#{@context}) | @namespace_tags
+      tags = %W(namespace:#{@namespace} context:#{@context}) | @namespace_tags
+      unless @current_sha.nil?
+        tags = %W(sha:#{@current_sha}) | tags
+      end
+      tags
```

Comment:
You can one-line this as `@current_sha.present? ? %W(sha:#{@current_sha}) : tags`